src/src/secret.py

```diff
@@ -1,8 +1,8 @@
 def getGithubKey():
-    return 'GHKEYXXX'
+    return '376d679b554ae9a9d474'
```

Comment:
Be careful and don't commit your GitHub App ID and Secret Key, review your changes before pushing them

Reply:
Thanks for pointing that out. I have corrected this.